collectors/plugins.d/plugins_d.h

```diff
@@ -26,10 +26,6 @@
 #define PLUGINSD_KEYWORD_FUNCTION               "FUNCTION"
 #define PLUGINSD_KEYWORD_FUNCTION_RESULT_BEGIN  "FUNCTION_RESULT_BEGIN"
 #define PLUGINSD_KEYWORD_FUNCTION_RESULT_END    "FUNCTION_RESULT_END"
-#define PLUGINSD_KEYWORD_GUID                   "GUID"
-#define PLUGINSD_KEYWORD_CONTEXT                "CONTEXT"
-#define PLUGINSD_KEYWORD_TOMBSTONE              "TOMBSTONE"
-#define PLUGINSD_KEYWORD_HOST                   "HOST"
```

Comment:
@stelfrag I also removed these. AFAICT, these were used only from the `metalog` functionality, which was removed a while ago, right?

Reply:
correct